packages/react-dom/src/client/validateDOMNesting.js

```diff
@@ -401,7 +401,7 @@ if (__DEV__) {
     return null;
   };
 
-  var didWarn = {};
+  let didWarn = {};
```

Comment:
This can probably be a `const`, since it's never re-assigned.

Reply:
fixed on 8436555fe9b076207d45cf8a30c8bb9f488a645c